analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -205,15 +205,11 @@ class SpdxDocumentFile(
     }
 
     /**
-     * Return null in case of "NONE" or "NOASSERTION" to fit with the standard process ( concluded manage by curations )
+     * Return the concluded license to be used in ORT's data model, which expects a not present value to be null
```

Comment:
Please squash this commit into the previous one, instead of addressing review comments in new commits.